google-api-go-generator/gen.go

```diff
@@ -719,6 +726,7 @@ func (a *API) GenerateCode() ([]byte, error) {
 	pn("const apiName = %q", a.doc.Name)
 	pn("const apiVersion = %q", a.doc.Version)
 	pn("const basePath = %q", a.apiBaseURL())
+	pn("const mtlsBasePath = %q", a.mtlsAPIBaseURL())
```

Comment:
```suggestion
	if mtlsBase := a.mtlsAPIBaseURL(); mtlsBase != "" {
		pn("const mtlsBasePath = %q", mtlsBase)
	}
```

("opts = append(opts, internaloption.WithDefaultMTLSEndpoint(mtlsBasePath))")

Reply:
done